rules.mk.j2

```diff
@@ -220,23 +220,27 @@ else
 	ci/vm.sh network_test | tee build/$@.log >/dev/$T
 endif
 
+ifneq ($(CI),drone)
+build/osie-aarch64.tar.gz: /proc/sys/fs/binfmt_misc/qemu-aarch64
+endif
 build/osie-aarch64.tar.gz: SED=/FROM/ s|.*|FROM multiarch/ubuntu-debootstrap:arm64-xenial|
-build/osie-x86_64.tar.gz: SED=
+build/osie-x86_64.tar.gz:  SED=
```

Comment:
extra space, no?

Reply:
lol, @mikemrm said the same thing. I did it on purpose to line up the SEDs in the Makefile, it was annoying me to be unaligned.